docker-plain/files/chart/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "chart.fullname" . }}-test-connection"
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: public.ecr.aws/docker/library/busybox
```

Comment:
Would it make sense to let this point to our BI docker repo proxy instead of aws? When is this test executed? On every helm deployment?

Reply:
oh wait, we can't put our inhouse URL here in github :)